.travis.yml

```diff
@@ -24,7 +28,10 @@ env:
   - PURE_RUBY=1
   - KITCHEN_SINK=1
 
-script: rake build && rake test:spec
+script: >
+  (bundle exec rake build) &&
+  (find . -not -path './vendor/*' -type f -iname '*.rb' -exec sed -i -z '1s/^\(\s*#\s*encoding[^\n]*\n\)\?/\1# frozen_string_literal: true\n/' '{}' +;) &&
```

Comment:
I'm thinking we can leave this out. It was certainly useful for testing! I'd rather see this magic comment committed to the files.

Reply:
The hack should be removed before merging IMO. The shift in stack traces would cause confusion.